expo-maps/ios/ExpoMaps/AppleMaps/AppleMapsView.swift

```diff
@@ -82,4 +80,16 @@ public final class AppleMapsView: UIView, ExpoMapView {
   func setShowLevelPicker(enable: Bool) {
     controls.setShowLevelPicker(enable: enable)
   }
+  
+  // imitating Google Maps zoom level behaviour
+  // based on https://gis.stackexchange.com/questions/7430/what-ratio-scales-do-google-maps-zoom-levels-correspond-to
+  func googleMapsZoomLevelToMeters(latitude: Double, zoom: Double) -> Double {
+    let metersPerPixel = 156543.03392 * cos(latitude * Double.pi / 180) / pow(2, zoom - 1)
+    return UIScreen.main.bounds.size.width * metersPerPixel
+  }
+  
```

Comment:
What do you think about using such function to make zoom levels working on `Apple Maps`? Should it stay like this although not 100% accurate or should we have other prop for passing distance in meters for `Apple Maps` (this might be inconvenient for the users)

Reply:
I think that this function is a better solution, even if it's not 100% accurate